sycl/test-e2e/ESIMD/accessor_load_store_stateless_64.cpp

```diff
@@ -0,0 +1,16 @@
+//===-accessor_load_store_stateless_64.cpp - DPC++ ESIMD on-device test-===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===-------------------------------------------------------------------===//
+// REQUIRES: gpu
+// UNSUPPORTED: gpu-intel-gen9 && windows
+// UNSUPPORTED: cuda || hip || esimd_emulator
+// RUN: %clangxx -fsycl -fsycl-esimd-force-stateless-mem %s -o %t.out
+// RUN: %GPU_RUN_PLACEHOLDER %t.out
+
+#define USE_64_BIT_OFFSET
```

Comment:
The changes look good.
Can you please do the test a bit stricter?
- create a 4Gig+128-bytes array; 
- write different values to beginning of array and to the end of it.
- on host: check that the values are written beyond 4Gig.
Such test would only work on PVC, I think.